charts/seed-monitoring/charts/core/charts/prometheus/rules-tests/kube-etcd3.rules.test.yaml

```diff
@@ -77,6 +87,31 @@ tests:
         pod: etcd
         job: kube-etcd3
       exp_annotations:
-        description: Etcd3 pod etcd has seen 81 proposal failures
-          within the last hour.
-        summary: High number of failed etcd proposals
\ No newline at end of file
+        description: Etcd3 pod etcd has seen 81 proposal failures within the last hour.
+        summary: High number of failed etcd proposals
+  - eval_time: 15m
+    alertname: KubeEtcdDeltaBackupFailed
+    exp_alerts:
+    - exp_labels:
+        service: etcd
+        severity: critical
+        type: seed
+        kind: Incr
+        job: kube-etcd3-backup-restore  
+        visibility: operator
+      exp_annotations:
+        description: No delta snapshot for the past 15 minutes.
+        summary: Etcd delta snapshot failure.     
+  - eval_time: 1455m
```

Comment:
```suggestion
  - eval_time: 24h
```

Reply:
same reason as above for keeping it 1455m instead of 24h